docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit/add-certificate-kubernetes.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Add cert-manager certificates to YugabyteDB Anywhere
+headerTitle: Add certificates
+linkTitle: Add certificates
+description: Add cert-manager certificates to YugabyteDB Anywhere.
+headcontent: Use your own certificates for encryption in transit
+menu:
+  preview_yugabyte-platform:
+    parent: enable-encryption-in-transit
+    identifier: add-certificate-4-kubernetes
+    weight: 20
+type: docs
+---
+
+{{<tabs>}}
+{{<tabitem href="../add-certificate-self/" text="Self-Signed" >}}
+{{<tabitem href="../add-certificate-ca/" text="CA-Signed" >}}
+{{<tabitem href="../add-certificate-hashicorp/" text="Hashicorp Vault" >}}
+{{<tabitem href="../add-certificate-kubernetes/" text="Kubernetes" active="true" >}}
+{{</tabs>}}
+
+For a universe created on Kubernetes, YugabyteDB Anywhere allows you to configure an existing running instance of the [cert-manager](https://cert-manager.io/) as a TLS certificate provider for a cluster.
```

Comment:
On kubernetes, the users also have the option of using self signed certificates and Hashicorp Vault certificates. Should we title this whole page "cert-manager" and clarify within the first para that it is only applicable to kubernetes?